### ISSUE TITLE
Unable to import openpgpjs in Netlify Function (AWS Lambda)

### ISSUE THREAD

**Issue by alxmhe** [openpgpjs/openpgpjs#1001]
I am trying to encrypt data from a serverless function running Node 10.x on AWS Lambda. I am using Netlify Functions for this Lambda.

When importing `openpgp`, I get this error :
```
Error: Cannot find module 'stream'
```

Here is the full code of the Netlify Function to test the issue :
```javascript
var openpgp = require('openpgp');

exports.handler = async function() {
  const message = "Lorem ipsum."
  const { privateKeyArmored, publicKeyArmored } = await openpgp.generateKey({
    userIds: [{ fileName: "" }],
    passphrase: '',
    numBits: 2048,
    curve: 'p256'
  })
  
  const ciphertext = await openpgp.encrypt({
    message: await openpgp.message.fromText(message),
    publicKeys: (await openpgp.key.readArmored(publicKeyArmored)).keys
  }).then(m => m && m.data)

  const privateKeyObj = (await openpgp.key.readArmored(privateKeyArmored)).keys[0]
  if (!privateKeyObj.isDecrypted())
    await privateKeyObj.decrypt("")
  const cleartext = await openpgp.decrypt({
    message: await openpgp.message.readArmored(ciphertext),
    privateKeys: [privateKeyObj],
  }).then(m => m && m.data)

  return {
    statusCode: 200,
    body: cleartext
  }
}
```

I originally thought the error came from the Node installation in the Lambda, but `stream` is accessible when required directly in the Function.

Any ideas on what would cause this issue?

**Comment by twiss** [comment]
Hey :wave: I think you need to specify the dependencies for Netlify somehow. See https://github.com/netlify/netlify-lambda/issues/142. It may have something to do with the Browserify `require` function being different from the normal `require`. If you `require('stream')` manually, is openpgp.js subsequently able to access it as well or not?

You could also try adding `stream` to your package.json dependencies, even though I think it should normally not be necessary.

**Comment by alxmhe** [comment]
Hi, I tried both of the above with no success. My script can access stream without any problem, but openpgpjs cannot, or probably its dependency web-stream.
Thank you for your help.

**Comment by tripodsan** [comment]
I see the same error when bundling my code with webpack.

**Comment by twiss** [comment]
This should be fixed by #1047 (in the upcoming v5 release).
